src/styles/vars.css

```diff
@@ -35,6 +34,34 @@
   --p-1\.5: 0.375rem;
   --p-2\.5: 0.625rem;
   --p-3\.5: 0.875rem;
+  --w-0: 0px;
+  --w-1: 0.25rem;
+  --w-2: 0.5rem;
+  --w-3: 0.75rem;
+  --w-4: 1rem;
+  --w-5: 1.25rem;
+  --w-6: 1.5rem;
+  --w-7: 1.75rem;
+  --w-8: 2rem;
+  --w-9: 2.25rem;
+  --w-10: 2.5rem;
+  --w-11: 2.75rem;
+  --w-12: 3rem;
+  --w-px: 1px;
+  --h-0: 0px;
+  --h-1: 0.25rem;
+  --h-2: 0.5rem;
+  --h-3: 0.75rem;
+  --h-4: 1rem;
+  --h-5: 1.25rem;
+  --h-6: 1.5rem;
+  --h-7: 1.75rem;
+  --h-8: 2rem;
+  --h-9: 2.25rem;
+  --h-10: 2.5rem;
+  --h-11: 2.75rem;
+  --h-12: 3rem;
+  --h-px: 1px;
```

Comment:
Pmst ma praegu panin need lihtsalt käsitsi. Mdea kas see on parim lahendus?

Reply:
Tostsin styles.css'is sest seal ei kirjutata neid genereerimisel yle. (review pris tehtud). vaatab mis edasi saab nendega.